pyhomematic/_devices.py

```diff
@@ -272,10 +283,7 @@ def stop(self):
     @property
     def working(self):
         """Return True of False if working or not"""
-        if self._PARENT:
-            return self._proxy.getValue(self._PARENT+':1', 'WORKING')
-        else:
-            return self.CHILDREN[1].getValue('WORKING')
+        return self._working
```

Comment:
Correct me if I'm wrong, but leaving the proxy-method away would result in a None return-value if there hasn't been a WORKING-event after starting pyhomematic.
My suggestion:
```python
if self._working == None:
    if self._PARENT:
        return self._proxy.getValue(self._PARENT+':1', 'WORKING')
    else:
        return self.CHILDREN[1].getValue('WORKING')
else:
    return self._working
```